src/BinDeps.jl

```diff
@@ -523,8 +524,14 @@ function run(s::SynchronousStepCollection)
     end
 end
 
-is_unix() && (make_command = `make -j8`)
-is_windows() && (make_command = `make`)
+const make_command =
```

Comment:
Not sure what is the purpose of `make_command` ?
It does not been exported and not used in the whole codebase either[1].
I guess it's just a stale variable.

[1].https://github.com/JuliaLang/BinDeps.jl/search?utf8=%E2%9C%93&q=make_command&type=